From b9b07b8e9ec4b54b2a2fdca7aae4aecd9aa86050 Mon Sep 17 00:00:00 2001
From: Daniel Borkmann <dborkman@redhat.com>
Date: Tue, 28 Jan 2014 09:54:36 -0500
Subject: [redhat] configs: add CONFIG_NETFILTER_XT_MATCH_CGROUP entry

Message-id: <97df516049f02f148d103e18e9d92c9095f0abb9.1390842850.git.dborkman@redhat.com>
Patchwork-id: 75436
O-Subject: [RHEL7 PATCH net 2/2] redhat: add CONFIG_NETFILTER_XT_MATCH_CGROUP entry
Bugzilla: 1005839
RH-Acked-by: Jiri Pirko <jpirko@redhat.com>
RH-Acked-by: David S. Miller <davem@redhat.com>
RH-Acked-by: Jesper Brouer <jbrouer@redhat.com>
RH-Acked-by: Florian Westphal <fwestpha@redhat.com>
RH-Acked-by: Thomas Graf <tgraf@redhat.com>

Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1005839
Upstream Status: none
Brew: http://brewweb.devel.redhat.com/brew/taskinfo?taskID=6954056
Tested: verified on beaker

Add the entry for CONFIG_NETFILTER_XT_MATCH_CGROUP.

Signed-off-by: Daniel Borkmann <dborkman@redhat.com>
Signed-off-by: Jiri Benc <jbenc@redhat.com>
Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/configs/generic/CONFIG_NETFILTER_XT_MATCH_CGROUP b/redhat/configs/generic/CONFIG_NETFILTER_XT_MATCH_CGROUP
new file mode 100644
index 0000000..d8a9d86
--- /dev/null
+++ b/redhat/configs/generic/CONFIG_NETFILTER_XT_MATCH_CGROUP
@@ -0,0 +1 @@
+CONFIG_NETFILTER_XT_MATCH_CGROUP=m
-- 
1.7.1